recipes/qtawesome/all/test_package/test_package.cpp

```diff
@@ -0,0 +1,11 @@
+#include "QtAwesome.h"
+#include <QApplication>
+
+int main(int argc, char *argv[]) {
+	QApplication app(argc, argv);
+	fa::QtAwesome awesome(&app);
+
+	awesome.initFontAwesome();
+
+    return EXIT_SUCCESS;
+}
```

Comment:
Indeed - the instantiation of `QApplication` will require Qt to locate the Cocoa platform plugin. In order to do so properly, we would need a qt.conf file or an environment variable (I believe `QT_PLUGIN_PATH` pointing to the location of the plugins). However this being a test_package being driven through command line only, we want to avoid instantiating a GUI window for the test.  I think removing the instance to `QApplication` is correct, otherwise we would need to set `QT_QPA_PLATFORM` environment variable, perhaps to `minimal` or `offscreen` (in addition to being able to locate the plugin).

Avoiding the instantiation of QApplication should be enough if there's a specific function we can directly test, as proposed by @uilianries.

Reply:
@jcar87 thank you for your detailed explanation! 💚